<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeRule</class>
 <widget class="QWidget" name="ChangeRule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="PushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="PrimaryPushButton" name="pushButton_7">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="StrongBodyLabel" name="label">
      <property name="text">
       <string>规则ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="LineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="StrongBodyLabel" name="label_2">
      <property name="text">
       <string>修改参数</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ComboBox" name="pushButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="StrongBodyLabel" name="label_3">
      <property name="text">
       <string>修改后的值</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="LineEdit" name="lineEdit_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
